Python/AWS testing Script/Readme.md

```diff
@@ -0,0 +1,29 @@
+## Aws Script for testing using moto (a AWS testing framework )
+
+This python script automatically generates a set of EC2 machine's with any demo ami Id that could be used in case of testing.
+
+## Installation
+
+First of all install [python]("https://www.python.org/downloads/") on your system.
```

Comment:
Installing python seems pretty redundant. 
Put all the packages to be installed in a `requirements.txt` file